hack/run-local.sh

```diff
@@ -7,8 +7,12 @@ oc scale --replicas 0 -n openshift-ingress-operator deployments ingress-operator
 
 IMAGE=$(oc get -n openshift-ingress-operator deployments/ingress-operator -o json | jq -r '.spec.template.spec.containers[0].env[] | select(.name=="IMAGE").value')
 RELEASE_VERSION=$(oc get clusterversion/version -o json | jq -r '.status.desired.version')
+NAMESPACE="openshift-ingress-operator"
+SHUTDOWN_FILE="${SHUTDOWN_FILE:-""}"
```

Comment:
@frobware if the `--shutdown-file` flag is not provided, "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem" is the default. An empty string is required for this flag to disable the file watcher, which is typically unneeded for local dev.

The operator does not start with the changes you propose due to this requirement:
```
$ make run-local
hack/run-local.sh
deployment.apps/cluster-version-operator scaled
deployment.apps/ingress-operator scaled
Image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:31e518340773eda69f7510ab0eeb64f2121b583f58c71d9621c899826895622e
Release version: 4.5.0-0.nightly-2020-05-04-113741
Namespace: openshift-ingress-operator
I0514 12:23:00.206315   36744 request.go:621] Throttling request took 1.010832189s, request: GET:https://api.dhansen.devcluster.openshift.com:6443/apis/user.openshift.io/v1?timeout=32s
2020-05-14T12:23:01.296-0700	INFO	operator.main	ingress-operator/start.go:107	using operator namespace	{"namespace": "openshift-ingress-operator"}
2020-05-14T12:23:01.426-0700	INFO	operator.dns	aws/dns.go:95	using region from shared config	{"region name": "us-west-2"}
2020-05-14T12:23:01.426-0700	ERROR	operator.main	ingress-operator/start.go:72	error starting	{"error": "failed to add file \"/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem\" to watcher: lstat /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem: no such file or directory"}
make: *** [run-local] Error 1
```

I have verified both use cases with my PR:

__filewatcher disabled__:
```
$ make run-local
hack/run-local.sh
deployment.apps/cluster-version-operator scaled
deployment.apps/ingress-operator scaled
Image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:31e518340773eda69f7510ab0eeb64f2121b583f58c71d9621c899826895622e
Release version: 4.5.0-0.nightly-2020-05-04-113741
Namespace: openshift-ingress-operator
I0514 12:25:19.468091   37082 request.go:621] Throttling request took 1.005706516s, request: GET:https://api.dhansen.devcluster.openshift.com:6443/apis/cloudcredential.openshift.io/v1?timeout=32s
2020-05-14T12:25:20.560-0700	INFO	operator.main	ingress-operator/start.go:107	using operator namespace	{"namespace": "openshift-ingress-operator"}
2020-05-14T12:25:20.688-0700	INFO	operator.dns	aws/dns.go:95	using region from shared config	{"region name": "us-west-2"}
2020-05-14T12:25:22.886-0700	INFO	operator.init.controller-runtime.metrics	metrics/listener.go:44	metrics server is starting to listen	{"addr": ":60000"}
<SNIP>
2020-05-14T12:25:27.271-0700	DEBUG	operator.init.controller-runtime.controller	controller/controller.go:282	Successfully Reconciled	{"controller": "ingress_controller", "request": "openshift-ingress-operator/default"}
```

__filewatcher enabled with local ca bundle__:
```
$ SHUTDOWN_FILE="/Users/daneyonhansen/tmp/tls-ca-bundle.pem" make run-local
hack/run-local.sh
deployment.apps/cluster-version-operator scaled
deployment.apps/ingress-operator scaled
Image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:31e518340773eda69f7510ab0eeb64f2121b583f58c71d9621c899826895622e
Release version: 4.5.0-0.nightly-2020-05-04-113741
Namespace: openshift-ingress-operator
I0514 12:26:40.858703   37244 request.go:621] Throttling request took 1.009838551s, request: GET:https://api.dhansen.devcluster.openshift.com:6443/apis/imageregistry.operator.openshift.io/v1?timeout=32s
2020-05-14T12:26:41.948-0700	INFO	operator.main	ingress-operator/start.go:107	using operator namespace	{"namespace": "openshift-ingress-operator"}
2020-05-14T12:26:42.076-0700	INFO	operator.dns	aws/dns.go:95	using region from shared config	{"region name": "us-west-2"}
2020-05-14T12:26:42.076-0700	INFO	operator.main	ingress-operator/start.go:163	watching file	{"filename": "/Users/daneyonhansen/tmp/tls-ca-bundle.pem"}
<SNIP>
2020-05-14T12:26:49.672-0700	DEBUG	operator.init.controller-runtime.controller	controller/controller.go:282	Successfully Reconciled	{"controller": "ingress_controller", "request": "openshift-ingress-operator/default"}
```

I did update `$NAMESPACE` so that the default value can be replaced, PTAL at commit `109772d`.

Reply:
> if the `--shutdown-file` flag is not provided, "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem" is the default. An empty string is required for this flag to disable the file watcher, which is typically unneeded for local dev.

Ah! Fair enough. Thanks for trying